ui/src/workflows/components/workflow-creator.tsx

```diff
@@ -21,6 +21,7 @@ export function WorkflowCreator({namespace, onCreate}: {namespace: string; onCre
     const [stage, setStage] = useState<Stage>('choose-method');
     const [workflow, setWorkflow] = useState<Workflow>();
     const [error, setError] = useState<Error>();
+    const isFullEditorDisabled = process.env.UI_DISABLE_FULL_EDITOR === 'true';
```

Comment:
Unfortunately, this isn't going to work, because this file is executed by the browser, and `process` is only valid in a NodeJS context. There's no way for the browser to know what environment variables are defined at the server level without an API call.

How did you test this? I ask because you should have seen a "process is not defined" error immediately on loading the workflow creator. If you didn't see an error, then something is probably wrong with your local environment.